app/views/event_sessions/event_session_comments/edit_comment.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#comment-<%=@event_session_comment.id%>').find('.media-heading').hide();
+$('#comment-<%=@event_session_comment.id%>').find('.comment-body').html("<%=  j(render partial:'event_sessions/event_session_comments/edit_comment') %>");
```

Comment:
The partial path looks inconsistent. In `routes.rb` I see `event_session_comment` resource is defined outside `event_session` scope but view path here indicates its within `event_session_scope`